R/add_isolate_info.R

```diff
@@ -0,0 +1,66 @@
+#' Adds info about isolates to tables of languages with glottocodes.
+#'
+#' @param Table data-frame. Required columns: level, Family_ID, Glottocode and Language_ID or Language_level_ID.
+#' @param  add_isolate_column Add a column to indicate whether a language is an isolate, or if it's a dialect of an isolate.
+#' @param set_isolates_family_as choice between c("themselves", "Isolate", "missing"). If set to 'themselves', the missing values for Family_ID for isolates is replaced with their glottocode, e.g. basq1248 gets the Family_ID basq1248. If 'Isolate' their Family_ID is set to the string 'Isolate'.
+#' @return Data-frame with desired modifications.
+#' @note If The current LanguageTable lacks the required columns, consider using a combination of the LanguageTable and ValueTable of glottolog-cldf.
+#' @export
+
+add_isolate_info <- function(Table = NULL,
+                             add_isolate_column = TRUE,
```

Comment:
would anyone ever call a function "add_isolate_info") and not `add_isolate_column`? can we just delete this argument?

Reply:
If you only want to manipulate Family_ID and don't care about having a column called "Isolate" you'd set this to FALSE.